bin/crew

```diff
@@ -1192,20 +1104,19 @@ def shrink_dir(dir)
         # 5. If the expansion doesn't error out then it is ok to copy
         # over the original. (This also lets us only avoid compressing
         # hard linked files multiple times.)
-        # Also disable for sommelier.elf and Xwayland.elf since upx
-        # breaks those binaries.
-        @execfiles = `find . -executable -type f ! \\( -name \"*.so*\" -o -name \"*.a\" -o -name \"Xwayland.elf\" -o -name \"sommelier.elf\" \\) -exec head -c4 {} 2>/dev/null \\; -printf ' %i %p\\n'| grep ^.ELF | sort -u -n -s -k2,2 | awk '{print $3}'`.chomp
-        unless @execfiles.to_s.empty?
+        @execfiles = `find . -executable -type f ! \\( -name '*.so*' -o -name '*.a' \\) | xargs -P8 -n1 sh -c '[ "$(head -c4 ${1})" = "\x7FELF" ] && echo ${1}' --`.chomp
```

Comment:
I think we still need the sommelier.elf and Xwayland.elf exclusion?

Reply:
Maybe it is better to handle it by the package itself? (by adding a `no_shrink` tag)

Reply:
Can you add those changes to those packages in this PR?

Reply:
I added these changes so this adjustment to crew is atomic. :)